src/main/java/com/minecolonies/coremod/commands/GetColonyAndCitizen.java

```diff
@@ -0,0 +1,216 @@
+package com.minecolonies.coremod.commands;
+
+import com.minecolonies.coremod.colony.ColonyManager;
+import net.minecraft.command.ICommandSender;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.UUID;
+
+/**
+ * Get the Colony ID and Citizen ID out of a command.
+ */
+
+public class GetColonyAndCitizen
+{
+
+    private static final String ONLY_NUMBERS            = "Please only use numbers for the %s ID!";
+    private static final String TOO_MANY_ARGUMENTS      = "Too many arguments!";
+    private static final String UNKNOWN_ERROR           = "Unknown Error!";
+    private static final String NOT_FOUND               = "%s not found!";
+    private static final String NO_COLONY               = "You haven't got a colony!";
+    private static final int    ONLY_NUMBERS_CODE       = -1;
+    private static final int    TOO_MANY_ARGUMENTS_CODE = -2;
+    private static final int    UNKNOWN_ERROR_CODE      = -3;
+    private static final int    NOT_FOUND_CODE          = -4;
+    private static final int    NO_COLONY_CODE          = -5;
+
+    private GetColonyAndCitizen()
+    {
+        throw new IllegalAccessError("Utility Class");
+    }
+
+    /**
+     * Getting the colony ID.
+     */
+    public static int getColonyId(@NotNull final ICommandSender sender, @NotNull final String... args)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document the parameter(s): sender, args [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this method return value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)